src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
```

Comment:
Would be more flexible to define max-width using calculations. In this case, we can say that the max width of the container is the width of the viewport - left - right padding.